subsys/usb/class/hid/core.c

```diff
@@ -122,6 +122,8 @@ static void hid_status_cb(enum usb_dc_status_code status, const u8_t *param)
 		case USB_DC_RESUME:
 			LOG_DBG("USB device resumed");
 			break;
+		case USB_DC_SOF:
```

Comment:
Following coding style we should add here LOG_DBG

Reply:
Yes, but SoF events happen very often (every 1ms for full speed).
If we add LOG_DBG here, the log will be cluttered with SoFs, rendering logs with debug level enabled unreadable.

Reply:
Ah, OK, we can leave it as is.